SimPEG/Examples/FLOW_Richards_1D_Celia1990.py

```diff
@@ -49,30 +51,30 @@ def run(plotIt=True):
     M = Mesh.TensorMesh([np.ones(40)])
     M.setCellGradBC('dirichlet')
     params = Richards.Empirical.HaverkampParams().celia1990
-    params['Ks'] = np.log(params['Ks'])
-    E = Richards.Empirical.Haverkamp(M, **params)
+    k_fun, theta_fun = Richards.Empirical.haverkamp(M, **params)
+    k_fun.KsMap = Maps.IdentityMap(nP=M.nC)
 
     bc = np.array([-61.5, -20.7])
     h = np.zeros(M.nC) + bc[0]
-    # bc = np.array([-20.7, -61.5])
-    # h = np.zeros(M.nC) + bc[1]
 
     def getFields(timeStep, method):
-        timeSteps = np.ones(360/timeStep)*timeStep
+        timeSteps = np.ones(int(360/timeStep))*timeStep
```

Comment:
For sure. I suppose timeStep should really be an int? `360//timeStep` would also work. This looks good.